src/main/java/org/benetech/servicenet/service/impl/OrganizationMatchServiceImpl.java

```diff
@@ -318,7 +317,7 @@ public void revertDismissOrganizationMatch(UUID id) {
 
             organizationMatchRepository.save(match);
 
-            conflictDetectionService.detectAsynchronously(match.getOrganizationRecord(), Collections.singletonList(match));
+            conflictDetectionService.reinstate(match);
```

Comment:
This potentially could be a good change, but I see here an edge case: when you reinstate a conflict between two records after one of them being completely changed and the conflict is obsolete and gives false/old info.